### ISSUE TITLE
Better Response from terminate()

### ISSUE THREAD

**Issue by wip-abramson** [OpenMined/PyDentity#65]
## Description
Currently the .terminate() function returns None. This is not very useful, lets make it better. Probably a boolean? Not sure, open to ideas.

**Comment by frogman** [comment]
Hi. Python returns none by default. Should we add 'return self' at the end and make use of it?

**Comment by wip-abramson** [comment]
Yeah maybe self, I hadn't thought of that. We should definitely return something I feel
